src/components/details-page/components/details-page/AccessRequestButton.tsx

```diff
@@ -0,0 +1,71 @@
+import Button from '@fellesdatakatalog/button';
+import React from 'react';
+import translations from '../../../../lib/localization';
+import { AccessRequest } from '../../../../types';
+import SC from './styled';
+import {
+  EventAction,
+  EventCategory,
+  trackSiteImproveEvent
+} from '../../../analytics-siteimprove/utils';
+
+const getAppliactionLink = async (location: string): Promise<string> => {
+  const accessRequestApi = `https://access-request.api.staging.fellesdatakatalog.digdir.no`;
+
+  const response = await fetch(
+    `${accessRequestApi}/access-request/${translations._language}${location}`
+  );
+
+  return response.text();
+};
+
+export function AccessRequestButton({
+  accessRequest,
+  entityId,
+  accessRequestUrl
+}: {
+  accessRequestUrl: string | undefined;
+  accessRequest: AccessRequest | undefined;
+  entityId: string | undefined;
+}) {
+  const trackAccessRequest = () => {
+    trackSiteImproveEvent({
+      category: EventCategory.DETAILS_PAGE,
+      action: EventAction.REQUEST_ACCESS,
+      label: entityId
+    });
+  };
+
+  const isEnturDataset = entityId === '8b285b05-dd33-31db-a1b6-0cf605bf05dd';
```

Comment:
Enturdatasettet for testing. Denne linjen kan fjernes når [accessRequestUrl](https://github.com/Informasjonsforvaltning/dcat-ap-no/issues/535) blir lagt til på datasettet